view.js

```diff
@@ -0,0 +1,32 @@
+const view = () => {
+
+    
+    const render = (posts) => {
+        let postContainer;
```

Comment:
No need to declare the variables outside the loop. Just declare and assign in the same place.